netlify.toml

```diff
@@ -7,45 +7,8 @@ ID = "Your_Site_ID"
 # unless otherwise overridden by more specific contexts.
 [build]
   base = ""
-  publish = "public/"
-  command = "bash build.bash"
+  publish = "site/"
+  command = "mkdocs build"
 
-# Have to set HUGO_VERSION for each environment otherwise we get the 0.17 default
-[context.production.environment]
-  HUGO_VERSION = "0.52"
-
-[context.deploy-preview.environment]
-  HUGO_VERSION = "0.52"
-
-[context.branch-deploy.environment]
-  HUGO_VERSION = "0.52"
-
-[[redirects]]
```

Comment:
these redirects should be kept and updated as appropriate. There's old links in the wild that will break if these aren't maintained.

Reply:
Redirects have been put back in.

Reply:
Awesome.